src/lib/cardFunctions.js

```diff
@@ -32,23 +32,16 @@ const suitChar = (suitChar) => {
 };
 //Displays the card's rank in a string
 const cardNumtoRank = (suitRank) => {
+  const characters = ["10", "J", "Q", "K", "A"];
```

Comment:
The first if checks if the suitRank is below or equal to 9 and returns the 3d char of the card string. Since 10 is 2 chars I added it to the special characters array.

Reply:
You can alternately check Number casting and it return NaN then it's not a number. Either way! It's good!